src/hapi/pipelines/database/org.py

```diff
@@ -150,6 +150,8 @@ def populate_multiple(self):
             )
             for org_data in self.data.values()
         ]
+        if len(org_rows) == 0:
+            return
```

Comment:
Is this unrelated?

Reply:
It's unrelated but came up while I was testing an individual country.  If there were no orgs being added the pipeline would fail completely instead of just not adding rows to the org table.